### PR TITLE
Add scripts for the operators bench

### LINKED ISSUE #316: Add benchmarking scripts for operators
The scripts will allow the replication of the result of section 6.1.2 of the paper.

### PR DESCRIPTION
**Pull request by interesaaat**
Closes #316. To merge after #328.

### CODE REVIEW

**Comment by ksaur** [comment]
Can you rebase (and remove html) and i'll check it?

File: hummingbird/ml/_topology.py [hunk 1]
```diff
@@ -39,15 +39,23 @@
 else:
     DataFrame = None
 
+if pandas_installed():
```
Comment (ksaur) [inline hummingbird/ml/_topology.py:42]: Something must be wrong with the rebase here.

Also i think there is some other problem with the prior merge... it seems that this [already was added...twice](https://github.com/microsoft/hummingbird/blob/master/hummingbird/ml/_topology.py#L37)

File: benchmarks/operators/score.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
+# Copyright (c) 2019, Microsoft CORPORATION. All rights reserved.
```
Comment (ksaur) [inline benchmarks/operators/score.py:1]: Does this file not need BSD/NVid license?  I know you wrote most of this file, just double checking

File: benchmarks/trees/score.py [hunk 3]
```diff
@@ -0,0 +1,257 @@
+# Copyright (c) 2019, Microsoft CORPORATION. All rights reserved.
```
Comment (ksaur) [inline benchmarks/trees/score.py]: (i think this is is only here from not rebase, but same question)

File: hummingbird/ml/_topology.py (hunk 1 above)
Comment (interesaaat) [reply hummingbird/ml/_topology.py:42]: oooh. Let me look into this.

File: benchmarks/operators/score.py (hunk 2 above)
Comment (interesaaat) [reply benchmarks/operators/score.py:1]: Yes this file contains only our stuff.

File: benchmarks/trees/score.py (hunk 3 above)
Comment (interesaaat) [reply benchmarks/trees/score.py]: Same answer :)

**Pull request merged**